app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/homescreen/HomeScreen.kt

```diff
@@ -0,0 +1,567 @@
+import hr.dice.filipbionda.tmdbpractice.ui.theme.black_100
+import hr.dice.filipbionda.tmdbpractice.ui.theme.grey_50
+import hr.dice.filipbionda.tmdbpractice.ui.theme.lightPurple
+import hr.dice.filipbionda.tmdbpractice.ui.theme.purple_36
+import hr.dice.filipbionda.tmdbpractice.ui.theme.secondaryColor
+import hr.dice.filipbionda.tmdbpractice.ui.theme.white
+import kotlinx.coroutines.launch
+
+private const val mockMovieUrl = "https://i.namu.wiki/i/__CjJoFpuzJXzMjM2DjQYvXCNf6UbCA_uaqgE5gubv80nATEJXEMwf01jV7kQnfkpREUrl2MEmR18H8_rUFAOg.webp"
+private const val mockSeriesUrl = "https://m.media-amazon.com/images/M/MV5BYWFjYmMxMjMtMGE2ZC00YWZhLTgzNDYtYTA3ODA2MDg2NTA4XkEyXkFqcGc@._V1_.jpg"
+private const val mockAnimeUrl = "https://u.livechart.me/anime/11850/poster_image/3531ac77e0fd178adc0875c1afa6ec16.webp/large.jpg"
+private const val mockSoapsUrl = "https://encrypted-tbn3.gstatic.com/images?q=tbn:ANd9GcR7HTzLmKUtvWDRCj_IJWxJw8CayAM8lZ0W3Ago_B33RWT0KfwB"
+private const val mockSpecialsUrl = "https://m.media-amazon.com/images/M/MV5BNzMyMTM1MjQxNF5BMl5BanBnXkFtZTgwMjY4NTE5NjE@._V1_.jpg"
+
+private val mockMovies =
+    listOf(
+        "movie1",
+        "movie2",
+        "movie3",
+        "movie4",
+        "movie5",
+        "movie6",
+    )
+
+private val mockSeries =
+    listOf(
+        "series1",
+        "series2",
+        "series3",
+        "series4",
+        "series5",
+        "series6",
+    )
+
+private val mockAnime =
+    listOf(
+        "anime1",
+        "anime2",
+        "anime3",
+        "anime4",
+        "anime5",
+        "anime6",
+    )
+
+private val mockSoaps =
+    listOf(
+        "soaps1",
+        "soaps2",
+        "soaps3",
+        "soaps4",
+        "soaps5",
+        "soaps6",
+    )
+
+private val mockSpecials =
+    listOf(
+        "specials1",
+        "specials2",
+        "specials3",
+        "specials4",
+        "specials5",
+        "specials6",
+    )
+
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier) {
+    var currentCategory by rememberSaveable {
+        mutableStateOf(ContentType.MOVIE)
+    }
+
+    val currentItems =
+        when (currentCategory) {
+            ContentType.MOVIE -> mockMovies
+            ContentType.SERIES -> mockSeries
+            ContentType.ANIME -> mockAnime
+            ContentType.SOAPS -> mockSoaps
+            ContentType.SPECIALS -> mockSpecials
+        }
+
+    val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to lightPurple,
+                0.6f to black_100,
+            ),
+        )
+
+    Box(
+        modifier =
+        modifier
+            .fillMaxSize()
+            .background(
+                brush = brush,
+            ),
+    ) {
+        HomeScreenHeadline(
+            modifier = Modifier.offset(y = dimensionResource(R.dimen.home_screen_headline_y_offset)),
+        )
+        HomeScreenSearchBar(
+            modifier = Modifier.zIndex(3f),
```

Comment:
Notice how you passed `modifier` for `HomeScreenChipGroup`. Either pass all of the parameters inlined or in a new line. Keep it consistent

Reply:
Done